Integrations/ThreatConnect/ThreatConnect.py

```diff
@@ -1299,6 +1299,195 @@ def tc_get_groups():
+        for attribute in data:
+            contents.append({
+                'ID': attribute.get('id'),
+                'Type': attribute.get('type'),
+                'Value': attribute.get('value'),
+                'DateAdded': attribute.get('dateAdded'),
+                'LastModified': attribute.get('lastModified'),
+                'Displayed': attribute.get('displayed')
+            })
+
+    else:
+        return_error(response.get('message'))
+
+    context = {
+        'TC.Group(val.ID && val.ID === obj.ID)': contents
+    }
+
+    return_outputs(
+        tableToMarkdown('Group Attributes', contents, headers, removeNull=True),
+        context
+    )
+
+
+def get_group_security_labels_request(group_type, group_id):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/groups/{}/{}/securityLabels'.format(group_type, group_id))
+
+    return tc.api_request(ro).json()
+
+
+def get_group_security_labels():
+    """
+    Retrieve a Group’s Security Labels
+    """
+    group_type = demisto.args().get('group_type')
+    group_id = int(demisto.args().get('group_id'))
+    contents = []
+    headers = ['Name', 'Description', 'DateAdded']
+    response = get_group_security_labels_request(group_type, group_id)
+    data = response.get('data', {}).get('securityLabel', [])
+
+    if response.get('status') == 'Success':
+        for security_label in data:
+            contents.append({
+                'Name': security_label.get('name'),
+                'Description': security_label.get('description'),
+                'DateAdded': security_label.get('dateAdded')
+            })
+
+    else:
+        return_error(response.get('message'))
+
+    context = {
+        'TC.Group(val.Name && val.Name === obj.Name)': contents
+    }
+
+    return_outputs(
+        tableToMarkdown('Group Security Labels', contents, headers, removeNull=True),
+        context
+    )
+
+
+def get_group_tags_request(group_type, group_id):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/groups/{}/{}/tags'.format(group_type, group_id))
+
+    return tc.api_request(ro).json()
+
+
+def get_group_tags():
+    """
+    Retrieve the Tags for a Group
+    """
+    group_type = demisto.args().get('group_type')
+    group_id = int(demisto.args().get('group_id'))
+    contents = []
+    context_entries = []
+    response = get_group_tags_request(group_type, group_id)
+    data = response.get('data', {}).get('tag', [])
+
+    if response.get('status') == 'Success':
+        for tags in data:
+            contents.append({
+                'Name': tags.get('name')
+            })
+
+            context_entries.append({
+                'ID': group_id,
+                'Name': tags.get('name')
+            })
+
+    else:
+        return_error(response.get('message'))
+
+    context = {
+        'TC.Group(val.Name && val.Name === obj.Name)': context_entries
```

Comment:
shouldnt this be by `ID` ?

Reply:
what ID? it returns only the name of the tag

Reply:
but you are filtering here by group name..